flow/util/distributed.py

```diff
@@ -0,0 +1,897 @@
+'''
+This scripts handles sweeping and tuning of OpenROAD-flow-scripts parameters.
```

Comment:
Does this handle Sweeping? If not, it would be better to mention only tuning. If it handle sweeping, please ignore this comment

Reply:
I have added sweeping as well. See [here](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/214/files#diff-054831345e7ef2722fe12b39ca1e9df603fb0297dc491658b5ec1c819f89e434R810-R833).